src/main/java/nextstep/ladder/controller/LadderController.java

```diff
@@ -12,11 +13,22 @@ public static void main(String[] args) {
 
         String peopleStr = inputView.inputPeople();
         People people = new People(peopleStr);
+
+        String resultStr = inputView.inputResults();
+        Results results = new Results(resultStr);
+
+
         int ladderHeight = inputView.inputLadderHeight();
 
-        Ladder ladder = new Ladder(people.getPeople().length, ladderHeight);
+        Ladder ladder = new Ladder(people.getPeople().size(), ladderHeight);
```

Comment:
바로 카운트를 출력해줘도 좋을 것 같아요!
```suggestion
        Ladder ladder = new Ladder(people.count(), ladderHeight);
```